src/components/ModelTableList/StatusGroup.js

```diff
@@ -104,7 +111,7 @@ function generateModelTableDataByStatus(groupedModels) {
       const cloud = generateCloudCell(model);
       const credential = getStatusValue(model.info, "cloud-credential-tag");
       const controller = getStatusValue(model.info, "controllerName");
-      const lastUpdated = getStatusValue(model.info, "status.since");
+      const lastUpdated = getStatusValue(model.info, "status.since")?.slice(2);
```

Comment:
Can you add a quick comment about what the format of the data looks like here before you slice it.